docs/developer-docs/docs/build-a-keychain/sdk/tutorial.md

```diff
@@ -0,0 +1,409 @@
+---
+sidebar_position: 2
+---
+
+# Building a Keychain service with Warden Protocol
```

Comment:
i can't remember if we agreed to capitalize all our terms such as
- keychain
- key request
- sign request
- key
but anyway i think we should have some consistency in the docs